lib/compiler/fmt.zig

```diff
@@ -276,7 +276,7 @@ fn fmtPathFile(
     const source_code = try std.zig.readSourceFileToEndAlloc(
         gpa,
         source_file,
-        std.math.cast(usize, stat.size) orelse return error.FileTooBig,
+        null,
```

Comment:
I don't understand this change. Doesn't this mean `fmtPathFile` will call `stat` twice instead of once?

Reply:
That is correct, I will remove this change (keeping the code as is, which will use `size_hint` instead and therefore not call stat twice.